packages/cloud_functions/cloud_functions_platform_interface/lib/src/platform_interface/platform_interface_https_callable.dart

```diff
@@ -58,14 +56,4 @@ abstract class HttpsCallablePlatform extends PlatformInterface {
   Future<dynamic> call([dynamic parameters]) {
     throw UnimplementedError('call() is not implemented');
   }
-
-  /// Returns the timeout for this instance.
-  Duration get timeout {
-    return options?.timeout;
-  }
```

Comment:
This timeout API isn't even surfaced on `cloud_functions` as a public API - I believe it was left overs from pre migration - I've now removed it.